IsIt5Yet.html

```diff
@@ -13,6 +13,15 @@
   </head>
   <body>
     <h1 class="sr-only">Is it 5 yet?</h1>
-    <p class="text-center transform translate-y-96 text-6xl text-white"></p>
+    <p
+      id="isIt5Yet"
+      class="text-center transform translate-y-96 text-6xl text-white"
```

Comment:
You can do a more responsive text centering. It should be in the center of the screen for all screen sizes. For longer messages  add text wrapping. For large screens it shouldn't be wider than 1/2 to 1/4 of the screen. For smaller screens it might be as wide as 3/4 the screen but never 100%.